products/appengine/terraform.yaml

```diff
@@ -93,6 +93,19 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         update_mask_fields:
           - "ssl_settings.certificate_id"
           - "ssl_settings.ssl_management_type"
+  ApplicationUrlDispatchRules: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{project}}"
+    import_format: ["{{project}}"]
+    mutex: "apps/{{project}}"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      custom_delete: templates/terraform/custom_delete/appengine.go.erb
```

Comment:
```suggestion
```

Reply:
I have added the same file at same location: templates/terraform/custom_delete/appengine.go.erb

Reply:
Ah, you're right. Unlike the `test_check_destroy`, that file is specific to `StandardAppVersion`. Can you delete it and remove the reference?

Reply:
ya, I replaced with ApplicationUrlDispatchRules. Not sure rest of the code in same file is correct or not.. just copied from other file.

Reply:
We don't want the custom delete function, and if we add one it will be different- I believe deletion was working correctly before.